compose/build.gradle.kts

```diff
@@ -0,0 +1,83 @@
+val composeBuild = gradle.includedBuild("support")
+fun Task.dependsOnComposeTask(name: String) = dependsOn(composeBuild.task(name))
+
+gradle.startParameter.excludedTaskNames.add(":support:compose:ui:ui-inspection:assembleInspectorJarRelease")
+gradle.startParameter.excludedTaskNames.add(":support:compose:ui:ui-inspection:dexInspectorRelease")
+
+// To show all projects which use `xxx` task, run:
+// ./gradlew -p frameworks/support help --task xxx
+
+tasks.register("publishComposeJb") {
+    dependsOnComposeTask(":compose:compiler:compiler:publishMavenPublicationToMavenRepository")
```

Comment:
Can we maintain this list in androidx repo, so that when refactoring they could be updated?

Reply:
We can't entirely, because it contains js tasks.

We can do it only for desktop, but I don't know what is the right place where it should be defined (maybe in root androidx/build.gradle, maybe in buildSrc).

Besides, most certainly we will be changing this ourselves, so better to keep this in compose-jb.

If androidx decides to publish MPP artifacts, they probably will write the scripts in a different way.

Reply:
Ok.